src/marqo/core/structured_vespa_index/structured_vespa_index.py

```
@@ -571,49 +571,6 @@ def _to_vespa_hybrid_query(self, marqo_query: MarqoHybridQuery) -> Dict[str, Any
 
         tensor_yql = f'select {select_attributes} from {self._marqo_index.schema_name} where {tensor_term}{filter_term}'
         lexical_yql = f'select {select_attributes} from {self._marqo_index.schema_name} where ({lexical_term}){filter_term}'
-        facet_queries = None
```

Maybe add a sanity check for these unsupported parameters to avoid silent failure?

---

The validation is already done at an early stage, see https://github.com/marqo-ai/marqo/blob/acd8f4e1507c4407b5f5a6d00700de1f7d39ddd8/src/marqo/core/search/hybrid_search.py#L147